src/wp-includes/rest-api/endpoints/class-wp-rest-widgets-controller.php

```diff
@@ -412,13 +412,15 @@ protected function save_widget( $request ) {
 			$id_base       = $parsed_id['id_base'];
 			$number        = isset( $parsed_id['number'] ) ? $parsed_id['number'] : null;
 			$widget_object = $wp_widget_factory->get_widget_object( $id_base );
+			$sidebar_id    = wp_find_widgets_sidebar( $id );
```

Comment:
Might be easier to pass `$sidebar_id` as an argument to `save_widget()` rather than calling this twice.